packages/maps/src/components/basic/GeoDistanceDropdown.js

```diff
@@ -122,13 +134,107 @@ class GeoDistanceDropdown extends Component {
 
 	componentWillUnmount() {
 		this.props.removeComponent(this.props.componentId);
+		this.props.removeComponent(this.internalComponent);
 	}
 
-	setReact(props) {
-		if (props.react) {
-			props.watchComponent(props.componentId, props.react);
+	static suggestionsQuery = (value, props) => {
```

Comment:
This function should be renamed to `defaultQuery`. We've made it static for SSR reasons. Ideally, it should be identical to that of DataSearch otherwise SSR will cause serious problems